online-shop/src/index.css

```diff
@@ -0,0 +1,102 @@
+.productDetails {
+ 
+    border-bottom: 1px solid #0d0d0e;
+    padding-bottom: 60px;
+    padding-left: 16px;
+    padding-top: 16px;
+    position: relative;
+
+ 
+}
+body{
+    background-color: lavender;
+}
+.productComponent{
+    width: 50%;
+    font-size: 15.5px;
+    padding: 30px;
+    text-align: left;
+    margin-top: 40px;
+    background-color: lavender;
+    
+}
+
+.productCategory{
+    font-size: 18px;
+    margin: 0;
+    line-height: 1cm;
+    
+}
+ 
+.productPrice{
+ 
+   /* color: #21c2f8;*/
+    /*font-family: Lato, sans-serif;*/
+    font-size: 25px;
+    line-height: 20px;
+   
+ 
+}
+.productPrice{
+    line-height: 1cm;
+}
+.productName {
+ 
+    font-size: 30px;
+    margin: 0;
+    line-height: 20px;
+    /*color:#000080	;*/
+   
+ 
+}
+.productDescription {
+ 
+    font-size: 15.5px;
+    line-height: 20px;
+    padding: 30px 14px 16px 19px;
+
+
+ 
+}
+.listItem{
+    font-size: 30.5px;
+    padding: 20px;
+    margin-left: 30px;
+}
+.myList{
+    background-color: lavender;
+    margin:50px 2px;
+    padding-left: 20px;
+   
+}
+#quantity{
+   text-align: right;
+   padding: 5px 20px;
+   margin:  0 0 0 5px;
+   font-size: 19.5px;
+}
+.QuantityButton {
+    margin: 0 0 0 5px ;
+    font-size: 19px;
+    cursor: pointer;
+    width: 29px;
+    height: 29px;
+    float: right;
+    text-align: center;
+   
+
+  }
+  .buttonWrapper{
+    float: right;
+  }
+  ul {
```

Comment:
Evita folosirea tag-urilor de html in fisierele de css, ca o sa ai situatii cand un fisier iti suprascrie stilurile din alt fisier. Evita si id-urile pe cat posibil. Foloseste clase.